### DISCUSSION TITLE
初来乍到，先放只流萤和纳西妲(≧∇≦)ﾉ

### DISCUSSION THREAD

**Discussion by HDGOLJ** [share121/inter-knot#3220] [其他]
![流萤](https://github.com/user-attachments/assets/61f1c163-6923-4f50-8065-34b34ade75c0)
![纳西妲](https://github.com/user-attachments/assets/89b575f8-6dd7-4bcf-a0ba-b24e15601495)

**Comment by qianchengmuxue-1997** [comment]
好看(♥∀♥)

**Comment by 123456bj** [comment]
好看<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/254352b8702fbdfe4b51919568b16f23_2308445851003216507.gif" alt="爱莉希雅-吃雪糕" width="75" >
